runtime-testsuite/pom.xml

```diff
@@ -119,6 +119,8 @@
                     <argument>${basedir}/test</argument>
                     <argument>-templates</argument>
                     <argument>${basedir}/resources/org/antlr/v4/test/runtime/templates</argument>
+                    <argument>-target</argument> <!-- TODO remove this before merging with upstream! -->
```

Comment:
Looks like there's already an item there for greping for pboyer, so either works for me.